source/ImageSet.cpp

```diff
@@ -146,18 +146,24 @@ namespace {
 				+ (ignored > 1 ? " frames" : " frame") + " ignored in total).");
 		}
 	}
+
+	string ToLowerCase(std::string str)
+	{
+		std::transform(str.begin(), str.end(), str.begin(), [](unsigned char c){ return std::tolower(c); });
+		return str;
+	}
```

Comment:
May I interest you in `Format::LowerCase`?
https://github.com/endless-sky/endless-sky/blob/f2bf985368a2df6a5f80e23cfdc8d8ccd075fd68/source/text/Format.cpp#L643-L649

Reply:
I knew we had this logic somewhere...